en/installation/debian.md

```diff
@@ -52,7 +52,7 @@ command curl -sSL https://rvm.io/pkuczynski.asc | gpg --import -
 curl -L https://get.rvm.io | bash -s stable
 ```
 
-then add rvm script source to user's bash (source /root/.bashrc) (this step is only necessary if you can't execute the rvm command)
+then add rvm script source to user's bash (source ~/.bashrc) (this step is only necessary if you can't execute the rvm command)
```

Comment:
I had forgotten that here the second "source" IMHO should be removed :thinking:. The original sentence was similar [1]. So now we should say "then add rvm script source to user's bash (~/.bashrc)".

The reason is, this sentence is related to the `source /usr/local/rvm/scripts/rvm` command, while the `source ~/.bashrc` command is related to the sentence "and finally, reload .bashrc to be able to run RVM". I'm not it I'm explaining it properly :sweat_smile:.

[1] https://github.com/consul/docs/pull/89/files#diff-85a70454ef88f40afe594805b3e88ca5L53